sites/all/themes/wille/template.php

```diff
@@ -144,6 +152,7 @@ function wille_preprocess_ting_object_cover(&$vars) {
         $type = reol_base_get_type_icon($entity->type);
         if ($type) {
           $vars['classes'] = array_merge($vars['classes'], _wille_type_icon_classes($type, $entity->reply->on_quota));
+          $vars['meta_for_labels'] = ['on_quota'=>$entity->reply->on_quota, 'type'=>$entity->type, 'id'->_entity->ding_entity_id];
```

Comment:
`'id'->_entity->ding_entity_id` does not look right. You're trying to access the property `_entity` on a string.

Do you mean `'id' => $entity->ding_entity_id`?

Remember to have all warnings enabled in PHP during development (https://www.drupal.org/docs/7/creating-custom-modules/show-all-errors-while-developing).